packages/payment-proxy-client/src/App.tsx

```diff
@@ -331,24 +323,25 @@ export default function App() {
                     alignItems="center"
                   >
                     <Typography variant="body2" color="text.secondary">
-                      {paymentChannelInfo?.Balance.Payer}
+                      {paymentChannelInfo?.Balance.RemainingFunds.toString()}
                     </Typography>
                     <PersonIcon />
                     <Slider
                       aria-label="Volume"
-                      value={
-                        paymentChannelInfo
-                          ? computePercentagePaid(paymentChannelInfo)
-                          : 0
-                      }
+                      value={Number(paymentChannelInfo?.Balance.PaidSoFar ?? 0)}
+                      min={0}
+                      max={Number(
+                        (paymentChannelInfo?.Balance.PaidSoFar ?? 0n) +
+                          (paymentChannelInfo?.Balance.RemainingFunds ?? 0n)
+                      )}
                       valueLabelDisplay="on"
                     />
                     <StorageIcon />{" "}
                     <Typography variant="body2" color="text.secondary">
-                      {paymentChannelInfo?.Balance.Payee}
+                      {paymentChannelInfo?.Balance.PaidSoFar.toString()}
                     </Typography>
                   </Stack>
-                  {paymentChannelId}
+                  {paymentChannelId.slice(0, 6) + "..."}
```

Comment:
Could be worth throwing this in a helper function; I bet we'll need to shorten channel ids in other places too.